backend/src/movements.rs

```diff
@@ -0,0 +1,80 @@
+use geo::{Euclidean, Length, Line, LineInterpolatePoint, Point, Polygon};
+use geojson::GeoJson;
+
+use crate::{
+    geo_helpers::{make_arrow, thicken_line},
+    Direction, IntersectionID, MapModel, Road,
+};
+
+impl MapModel {
+    pub fn get_movements(&self, i: IntersectionID) -> GeoJson {
```

Comment:
No biggie so take it or leave it: `movement_arrows` or `movement_labels` might be a better name, since these seem to be intended only as UI affordances, vs some business-logic struct.

Reply:
To minimize churn, I'll leave this for now. Right now it's a debug mode, but that will probably turn into some kind of "edit turn restrictions at an intersection" mode, and the movements sent over will have a semantic ID that the backend knows how to edit.